File: Medium-Questions/06-circle-Area/C/Six.c
#include <stdio.h>

float pi = 3.14159265359;

int main(void)
{
    // get input form user
    int r;
    printf("Enter radius: ");
    scanf("%i",&r);

    // calculate area
    float n = ( (pi) * (r ** 2));

    printf("circle of circle is %f ",n);
    return 0;
}

File: Medium-Questions/06-circle-Area/Python/six.py
import sys
import math

r = float(input("Enter r: "))

# formula for calculate radius of circle 
# c = 2pr
# https://www.wikihow.com/Calculate-the-Radius-of-a-Circle


p = math.pi

answer = p * (r ** 2)

print("Area: ",answer)

sys.exit(0)
